DESCRIPTION

```diff
@@ -9,6 +9,7 @@ Description: Functions to assist in R programming, including:
   - define macros ('defmacro'),
   - detect odd and even integers ('odd', 'even'),
   - convert strings containing non-ASCII characters (like single quotes) to plain ASCII ('ASCIIfy'),
+  - convert numbers of base 10 to representations of any other base (e.g. dual, octal or hexadecimal)
```

Comment:
A better description would be 
> - convert a base-10 number to a sequence of digits in other bases (e.g, binary, octal, hexadecimal).

Reply:
Only read this better decription now. Done.